web/src/rest.ts

```diff
@@ -219,8 +219,8 @@ const dandiRest = new Vue({
       return data;
     },
     async unembargo(identifier: string) {
-      // TODO: implement this once the server endpoint is available
-      return identifier;
+      const { data } = await client.post(`dandisets/${identifier}/unembargo/`);
+      return data;
```

Comment:
This isn't exactly wrong but that endpoint doesn't return any body, just status codes. I don't know if we need to change this but just wanted to point it out.

Reply:
https://github.com/dandi/dandi-archive/pull/883/commits/653f249b7d1829cd8168b5aec7d19390323a85e5